### PR TITLE
[SIG-1782] Toelichting line-height

### PR DESCRIPTION
**Pull request by janjaap**
This PR sets the line-height for the status change feedback block.

Before:
Screenshot 2019-10-31 at 12 30 11

After:
Screenshot 2019-10-31 at 12 29 12

### CODE REVIEW

**Comment by jasperswart** [comment]
moeten we niet eerst een Alert component maken in ASC?

https://designsystem.amsterdam.nl/7awj1hc9f/p/901ede

**Comment by janjaap** [comment]
@jasperswart That seems to be out-of-scope for this PR; the corresponding ticket didn't mention anything of that kind.

File: src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js [hunk 1]
```diff
@@ -78,8 +102,18 @@ class StatusForm extends React.Component { // eslint-disable-line react/prefer-s
...
+        prevProps.patching && prevProps.patching.status,
+        this.props.patching && this.props.patching.status
+      ) &&
+      this.props.patching.status === false
+    ) {
+      const hasError =
+        (this.props.error &&
+          this.props.error.response &&
+          !this.props.error.response.ok) ||
+        false;
```
Comment (spaces-ga) [inline src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js]: Not changed by this PR, but this is already a boolean comparison

File: src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js [hunk 2]
```diff
@@ -149,26 +184,33 @@ class StatusForm extends React.Component { // eslint-disable-line react/prefer-s
...
                     {warning}
-                  </div>
-                  <div className="notification notification-red" data-testid="statusFormError">
-                    {error && error.response && error.response.status === 403 ? 'Je bent niet geautoriseerd om dit te doen.' : '' }
-                    {error && error.response && error.response.status !== 403 ? 'De gekozen status is niet mogelijk in deze situatie.' : '' }
-                  </div>
+                  </Notification>
+
+                  <Notification warning data-testid="statusFormError">
+                    {error && error.response && error.response.status === 403
```
Comment (spaces-ga) [inline src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js]: Could we turn these codes into constants? `HTTP_FORBIDDEN`

Could also replace conditional (?) by short-circuit evaluation. Removing the need for the empty string.

File: src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js [hunk 3]
```diff
@@ -149,26 +184,33 @@ class StatusForm extends React.Component { // eslint-disable-line react/prefer-s
...
+                  </Notification>
 
                   <StyledButton
                     data-testid="statusFormSubmitButton"
                     variant="secondary"
                     disabled={invalid}
                     type="submit"
-                    iconRight={patching.status ? <StyledSpinner /> : null}
-                  >Status opslaan</StyledButton>
+                    iconRight={patching.status ? <StyledSpinner /> : null}>
```
Comment (spaces-ga) [inline src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js]:
```js
patching.status && <StyledSpinner />
```

File: src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js (hunk 2 above)
Comment (janjaap) [reply src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js]: Good point; I'll make a ticket for the constants suggestion.

**Pull request merged**